Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/AnimationsWatched.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class AnimationsWatched
+    {
+        public int AnimationId { get; set; }
+        public int UserId { get; set; }
+        public  Animation animation{ get; set; }
```

Comment:
Here you're using PascalCase then you're using lower case for animation and user because you cant give them the same name I get it, but use different names then so that you stick to one standard